yb-voyager/cmd/importData.go

```diff
@@ -915,7 +915,7 @@ func doOneImport(task *SplitFileImportTask, connPool *tgtdb.ConnectionPool) {
 	}
 }
 
-func executeSqlFile(file string) {
+func executeSqlFile(file string, objType string) {
 	log.Infof("Execute SQL file %q on target %q", file, target.Host)
 	connectionURI := target.GetConnectionUri()
```

Comment:
Define a separate function to wrap the code to instantiate a new connection and set schema if required. We can call it something like `newTargetConn()`.
See the other comment to know how can we use it to have a new connection after each error.

Reply:
done